utils/scripts/terraform/openstack/provision.py

```
@@ -115,6 +116,12 @@ def set_config(self):
         os.makedirs(dst_dir, exist_ok=True)
         shutil.move(output_file, os.path.join(dst_dir, output_file))
 
+    def get_ocm_latest_tag(self):
+        resp = requests.get("https://api.github.com/repos/openshift-online/ocm-cli/releases/latest")
+        resp_dict = resp.json()
+
+        return resp_dict["tag_name"]
```

IMO, we should have a `try/expect` statement so that we fail early or return a default value.

Also, add docstrings of the module.

---

@psathyan Thank you. I have incorporated the docstring and the try/except block with the latest commit.
Please review